servo/api.py

```diff
@@ -188,7 +191,7 @@ async def report_progress(self, **kwargs) -> None:
             raise servo.errors.UnexpectedEventError(status.reason)
         elif status.status == OptimizerStatuses.cancelled:
             # Optimizer wants to cancel the operation
-            raise servo.errors.EventCancelledError(status.reason)
+            raise servo.errors.EventCancelledError("Command cancelled")
```

Comment:
Please update this to be `(status.reason or "Command cancelled")` so the backend status is communicated when present

Reply:
Good call, thank you